content/blog/custom-docker-images-for-emr-serverless.md

```diff
@@ -0,0 +1,199 @@
+
+### **Reference links to AWS documentations:**
+
+* *Steps to build custom docker image for EMRServerless:* <https://aws.amazon.com/blogs/big-data/add-your-own-libraries-and-application-dependencies-to-spark-and-hive-on-amazon-emr-serverless-with-custom-images/>
+* *How to pull docker image from AWS ECR registry:* <https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-pull-ecr-image.html>
+* *How to authorize AWS ECR registry with docker client:* <https://docs.aws.amazon.com/AmazonECR/latest/userguide/registry_auth.html>
+* *How to push docker image to AWS ECS registry:* <https://docs.aws.amazon.com/AmazonECR/latest/userguide/docker-push-ecr-image.html>
+* *AWS EMRServerless CLI reference:* <https://docs.aws.amazon.com/cli/latest/reference/emr-serverless/index.html>
+* *How to customize image for multi CPU architecture:* <https://docs.aws.amazon.com/emr/latest/EMR-on-EKS-DevelopmentGuide/docker-custom-images-steps.html>
+
+<!--EndFragment-->
+
+## **About the Authors:**
+
+![](/img/niranjan_2.jpg)
+
+<!--StartFragment-->
+
+D R Niranjan is a Senior software/Cloud application developer at HPE. Experienced in developing software and cloud applications for HPE Servers and Storage systems. Technically skilled in Python, Java, Scala, Spark programming, AWS, Dockers and Kubernetes, PgSQL/RDBMS. You can have a look at his LinkedIn profile here: <https://www.linkedin.com/in/niranjan-d-r/>
+
+<!--EndFragment-->
+
+![](/img/sagar-pic_2.jpg "sagar-nyamagouda@hpe.com")
+
+Sagar Nyamagouda holds a  B.E(Information Science and Engineering) from BMS College of Engineering (BMSCE), Bengaluru and M.Tech in Software Systems from BITS Pilani. He is an experienced R&D Engineer working on Big Data Technologies and building AI/ML pipelines for give real time insights to customers. An AI/ML enthusiast. Currently working with HPE enabling advanced insights for DSCC - an HPE Flagship cloud offering.   LinkedIn Profile: www.linkedin.com/in/sagarny
+
+![](/img/chirag_2.jpg "chirag.talreja@hpe.com")
+
+Graduate from BITS Pilani, Pilani . Currently working as a Cloud Developer in HPE with over 6 years of experience in designing, developing, and implementing complex data processing pipelines . Experienced in microservice architecture,  big data tech stack like Apache Spark , Spark Streaming , SQL/NoSQL databases, Kafka, Core Java, Scala, python with  a good understanding of AWS platform. His linkedIn profile is available @ <https://www.linkedin.com/in/chiragtalreja29/>
+
+![](/img/chinmay_2jpg.jpg "chinmay.chaturvedi@hpe.com")
+
+Hello Everyone!! I am Chinmay , currently working in HPE as a Cloud Engineer. I have expertise in  various technologies of Big  Data Processing  including building ML/AI platforms, data engineering and processing out of which most recently are related to spark, AWS , SQL etc. Apart from work I like playing badminton and fond of going on trips. LinkedIn Profile: <https://www.linkedin.com/in/chinmay-chaturvedi-707886138>
```

Comment:
Because the rest of the bios were significantly more formal (and less familiar than this one), I'd recommend rewriting this so it has a parallel structure to the rest. A proposed rewrite would be:
"Chinmay currently works at HPE as a cloud engineer.  He has expertise in various Big Data Processing technologies, including the building of ML/AI platforms, data engineering and processing (most recently related to Spark, AWS, SQL, etc.). Please view his LinkedIn profile here.